kirby.php

```diff
@@ -491,20 +509,23 @@ public function extensions() {
    */
   public function models() {
 
-    if(!is_dir($this->roots()->models())) return false;
+    $roots   = $this->modules()->allRoots('models', $this->roots()->models());
```

Comment:
The `return false` from the old version got lost here. For compatibility, the method should probably return false if it checked all roots and none existed.